packages/markdown-slate/src/slateToCiceroMarkDom.js

```diff
@@ -32,6 +32,19 @@ function slateToCiceroMarkDom(document) {
     // convert the value to a plain object
     const json = JSON.parse(JSON.stringify(document));
     _recursive(result, json.nodes);
+    const isLastNodePara = result.nodes[result.nodes.length - 1]
```

Comment:
This should be made a bit more defensive so it does not fail when the array is empty.

Reply:
Also this should only apply when the last paragraph is preceded by a clause node.

Reply:
Minor code style: `result.nodes[result.nodes.length - 1]` can be put in a variable, somewhat more concise.